### PR TITLE
Update the appstudio user actions role check with the new requirements.

### PR DESCRIPTION
**Pull request by metlos**
This is an e2e test for https://github.com/codeready-toolchain/host-operator/pull/626

### CODE REVIEW

File: testsupport/tiers/checks.go [hunk 1]
```diff
@@ -1057,6 +1062,7 @@ func appstudioUserActionsRole() namespaceObjectsCheck {
 					Resources: []string{"results", "records"},
 					Verbs:     []string{"get", "list"},
 				},
+                           },
```
Comment (MatousJobanek) [inline testsupport/tiers/checks.go:1065]: ?

**Review by MatousJobanek — COMMENTED** [review]
please update also the expected number of rules: https://github.com/codeready-toolchain/toolchain-e2e/blob/27f03fc67a2a464b07b9d362bd8a66e627014150/testsupport/tiers/checks.go#L1022

**Comment by alexeykazakov** [comment]
Closing in favor of https://github.com/codeready-toolchain/toolchain-e2e/pull/542

**Pull request closed without merging**